README.md

```diff
@@ -39,16 +39,16 @@ After instantiation, a [public API](#public-api) is available with a limited num
     venueId: '10',
     configurationId: '1046',
     ticketGroups: [{
-      tevo_section_name: 'Upper 1234',
-      price: 100
+      tevo_section_name: 'upper end zone 232',
```

Comment:
@jwcobb Believe this PR is good, but wanted to make one comment that this `tevo_section_name` attr will be the original section name if no match is found, just a FYI.  This attribute is meant to be a lookup and not a displayable entry.

Reply:
Right. I just like when docs are as close to real-world as possible.

Reply:
Good point. The client can handle non-lower cased names, but that's not intended to be the happy path.